src/Service/Program.cs

```diff
@@ -205,5 +213,17 @@ private static void AddConfigurationProviders(
                 .AddEnvironmentVariables(prefix: FileSystemRuntimeConfigLoader.ENVIRONMENT_PREFIX)
                 .AddCommandLine(args);
         }
+
+        private static bool ValidateAspNetCoreUrls()
+        {
+            if (Environment.GetEnvironmentVariable("ASPNETCORE_URLS") is not string value)
+            {
+                return true; // If the environment variable is missing, then it cannot be invalid.
+            }
+
+            return value
+                .Split(new[] { ',', ';' }, StringSplitOptions.RemoveEmptyEntries)
+                .All(x => Uri.TryCreate(x.Trim(), UriKind.Absolute, out _));
```

Comment:
Could someone explain what is the purpose of the different types of UriKind?

Reply:
> Could someone explain what is the purpose of the different types of `UriKind`?

It is to indicate if you are using an "absolute" URL, which means you are including the entire URL, or if you are using a "relative" URL, which would then be concatenated onto the end of the "base" URL.

ie: "http://foo.com" vs "foo.com" where we have a base URL of "http://"